src/main/scala/craft/Configs.scala

```diff
@@ -66,6 +66,7 @@ object ConfigBuilder {
     }) ++ nastiTLParams(id)
   def nastiTLParams(id: String): Config = new Config(
     (pname, site, here) => pname match {
+      case XBarUsePortQueues => true
```

Comment:
@stevobailey I don't know if you think this is OK. It is kind of shady, but all the configs call it. It doesn't really belong here, but I think it requires the least amount of fiddling with other configs.